tests/bdd/features/volume/nexus-info/feature.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Persistent Nexus Info
+
+  Background:
+    Given an existing volume
+
+  Scenario: publishing a volume
+    Given a volume that is not published
+    When the volume is published
+    Then the nexus info structure should be present in the persistent store
+
+  Scenario: unpublishing a volume
+    Given a volume that is published
+    When the volume is unpublished
+    Then the nexus info structure should be present in the persistent store
```

Comment:
Why do we not delete the nexus info when we unpublish, on re-publish we anyway delete it so can we not delete it at this point itself? Just for my understanding.

Reply:
Good question. If we delete it when unpublishing, we lose the information about which children are healthy. Then, when we try to publish again we won't know which children should be used, so we end up adding all of them. This can lead to data inconsistency between children if one of them was previously faulted.

Reply:
Okay. Thanks for explaining.